### ISSUE TITLE
AutocompletePageTest fails in Edge

### ISSUE THREAD

**Issue by warpech** [StarcounterSamples/KitchenSink#52]
@aloscha noticed that sometimes one test in AutocompletePageTest fails in Edge. I have tried on my machine and actually two tests fail for me.

@joozek78 could you please take a look? Some notes:

> 1) Ignored : KitchenSink.Tests.Tests.AutocompletePageTest("edge").FillPlaceNameT
> henBlurExpectHintsGone
> Blur event does not fire in edge under selenium

This is not an excuse. Can't we just simulate blur by clicking on anything else? (`body` for instance)

> Element is obscured (WARNING: The server did not provide any stacktrace information)

I think that this error would not appear if you used `ClickUsingMouse` instead of `Click`.

---

Full error report:

``` text
c:\www\StarcounterSamples\KitchenSink>packages\NUnit.ConsoleRunner.3.2.0\tools\n
unit3-console.exe test\KitchenSink.Tests\KitchenSink.Tests.csproj /config:Debug

NUnit Console Runner 3.2.0
Copyright (C) 2016 Charlie Poole

Runtime Environment
   OS Version: Microsoft Windows NT 10.0.10586.0
  CLR Version: 4.0.30319.42000

Test Files
    test\KitchenSink.Tests\KitchenSink.Tests.csproj


Tests Not Run

1) Ignored : KitchenSink.Tests.Tests.AutocompletePageTest("edge").FillPlaceNameT
henBlurExpectHintsGone
Blur event does not fire in edge under selenium

Errors and Failures

1) Failed : KitchenSink.Tests.Tests.AutocompletePageTest("edge")
One or more child tests had errors

2) Error : KitchenSink.Tests.Tests.AutocompletePageTest("edge").FillCountryNameT
henSelectCountry
System.InvalidOperationException : Element is obscured (WARNING: The server did
not provide any stacktrace information)
Command duration or timeout: 16 milliseconds
Build info: version: '2.52.0', revision: '4c2593c', time: '2016-02-11 19:06:42'
System info: host: 'DESKTOP-ITUP0CI', ip: '192.168.0.102', os.name: 'Windows 10'
, os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.edge.EdgeDriver
Capabilities [{acceptSslCerts=true, browserVersion=25.10586.0.0, platformVersion
=10, browserName=MicrosoftEdge, takesScreenshot=true, pageLoadStrategy=normal, t
akesElementScreenshot=true, platformName=windows, platform=ANY}]
Session ID: 6EE009DF-C588-46FE-B5B5-1D24D0879060
   at OpenQA.Selenium.Remote.RemoteWebDriver.UnpackAndThrowOnError(Response erro
rResponse)
   at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecu
te, Dictionary`2 parameters)
   at OpenQA.Selenium.Remote.RemoteWebElement.Click()
   at KitchenSink.Tests.Tests.AutocompletePageTest.FillCountryNameThenSelectCoun
try() in c:\www\StarcounterSamples\KitchenSink\test\KitchenSink.Tests\Tests\Auto
completePageTest.cs:line 49

3) Error : KitchenSink.Tests.Tests.AutocompletePageTest("edge").FillProductNameT
henSelectProduct
System.InvalidOperationException : Element is obscured (WARNING: The server did
not provide any stacktrace information)
Command duration or timeout: 16 milliseconds
Build info: version: '2.52.0', revision: '4c2593c', time: '2016-02-11 19:06:42'
System info: host: 'DESKTOP-ITUP0CI', ip: '192.168.0.102', os.name: 'Windows 10'
, os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.edge.EdgeDriver
Capabilities [{acceptSslCerts=true, browserVersion=25.10586.0.0, platformVersion
=10, browserName=MicrosoftEdge, takesScreenshot=true, pageLoadStrategy=normal, t
akesElementScreenshot=true, platformName=windows, platform=ANY}]
Session ID: 108FF432-6CEA-4730-A085-C9D300017137
   at OpenQA.Selenium.Remote.RemoteWebDriver.UnpackAndThrowOnError(Response erro
rResponse)
   at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecu
te, Dictionary`2 parameters)
   at OpenQA.Selenium.Remote.RemoteWebElement.Click()
   at KitchenSink.Tests.Tests.AutocompletePageTest.FillProductNameThenSelectProd
uct() in c:\www\StarcounterSamples\KitchenSink\test\KitchenSink.Tests\Tests\Auto
completePageTest.cs:line 64

4) SetUp Failed : KitchenSink.Tests.Tests.RedirectPageTest("chrome")
One or more child tests had errors

5) SetUp Failed : KitchenSink.Tests.Tests.RedirectPageTest("edge")
One or more child tests had errors

6) SetUp Failed : KitchenSink.Tests.Tests.RedirectPageTest("firefox")
One or more child tests had errors

Test Run Summary
  Overall result: Failed
  Test Count: 72, Passed: 60, Failed: 11, Inconclusive: 0, Skipped: 1
    Failed Tests - Failures: 0, Errors: 11, Invalid: 0
    Skipped Tests - Ignored: 1, Explicit: 0, Other: 0
  Start time: 2016-06-03 09:24:59Z
    End time: 2016-06-03 09:30:30Z
    Duration: 330.760 seconds

Results (nunit3) saved as TestResult.xml
```

**Comment by aloscha** [comment]
Yeah, I have the same what mention @warpech, but also test with button fails for me:

      Test Name:	ButtonPageTest_DisabledButton
      Test FullName:	KitchenSink.Test.ButtonPageTest("edge").ButtonPageTest_DisabledButton
      Test Source:	C:\Users\aloscha\Documents\KitchenSink\test\KitchenSink.Tests\Tests\ButtonPageTest.cs : line 123
      Test Outcome:	Failed
      Test Duration:	0:00:14.839

      Result StackTrace:	
      at OpenQA.Selenium.Support.UI.DefaultWait`1.ThrowTimeoutException(String exceptionMessage, Exception lastException)
      at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition)
      at KitchenSink.Test.ButtonPageTest.ButtonPageTest_DisabledButton() in C:\Users\aloscha\Documents\KitchenSink\test\KitchenSink.Tests\Tests\ButtonPageTest.cs:line 135
      Result Message:	OpenQA.Selenium.WebDriverTimeoutException : Timed out after 10 seconds

**Comment by warpech** [comment]
Closing as no longer relevant after #83. 

Tests are still failing in Edge, but that's another story (https://github.com/StarcounterSamples/KitchenSink/issues/59)
